ImageUpdate/test_image.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+source ./common.sh
+set -euf -o pipefail
+IMAGE=${1}
+VMNAME="${IMAGE}-test"
+# Timeout in seconds
+TIMEOUT=300
+sourcerc
+function vmIsActive() {
+    if [ "$(openstack server show "${VM_ID}" -f json | jq -r '.status')" == "ACTIVE" ]; then
+        return 1
+    fi
+    return 0
+}
+function catchExit(){
+    if [[ "$VM_ID" == "" ]];then
+        error "Error creating server for $IMAGE"
+    fi
+}
+if [[ "$IMAGE" =~ "Windows" ]];then
+    warn "skipped $IMAGE: Can't test Windows"
+    exit 0
+fi
+
+trap catchExit ERR 
+SEC_GROUP="$(openstack security group list --project admin -f json | jq -r '.[] | select(.Name == "default") | .ID')"
+VM_ID="$(openstack server create --flavor 3 --image "${IMAGE}" \
```

Comment:
maybe its better to define the flavor in a variable, also I think you can use the flavor's name instead of the ID from the flavor's variable